modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -38,27 +38,23 @@ Redpanda is designed to maximize throughput by xref:get-started:architecture.ado
 - **Loss of CPU cache affinity**: As processes move across cores, they lose the cache locality, leading to increased cache misses and higher latency.
 - **Scheduling latency**: Dynamic CPU allocation can introduce delays as the scheduler determines which core a process should run on, based on current load and availability.
 
-Enabling CPU pinning for Redpanda in Kubernetes requires the following:
-
-- *Static CPU Manager policy*: To enable CPU pinning in Kubernetes, you must enable the https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy[`static` CPU Manager policy], which bypasses the default CPU allocation behavior managed by the CFS.
-+
-When the `static` policy is enabled, each Pod is assigned specific CPU cores, and these assignments remain constant throughout the Pod's lifecycle. This fixed allocation is crucial for Redpanda in production, because it maintains CPU cache affinity. Cache affinity refers to the performance benefit that arises when a process is able to repeatedly access data from the same CPU cache, rather than re-fetching the data from main memory or a distant cache associated with another CPU core.
-- *Overprovisioning disabled (default)*: When overprovisioning is disabled in Redpanda, each thread runs on a dedicated core for maximum performance.
-- *Guaranteed quality of service for all Pods that run Redpanda*: Only containers that are both part of a https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#guaranteed[`Guaranteed` Pod] and have integer CPU requests are assigned exclusive CPUs.
+To enable CPU pinning in Kubernetes, you must enable the https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy[`static` policy] for the CPU manager. With this policy enabled, the CPU Manager controls a shared pool of CPUs. Initially, this shared pool contains all the CPUs in the worker node. However, when a container is created with an integral CPU request in a https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[`Guaranteed` Pod], CPUs for that container are removed from the shared pool and assigned exclusively to that container for its lifetime. Other containers are migrated off these exclusively allocated CPUs.
 
 === Enable CPU pinning
 
+To enable CPU pinning for Redpanda in Kubernetes:
+
 . Ensure that your worker nodes have enough CPU cores to allocate to Redpanda. Sharing cores with other applications can negate the benefits of CPU pinning.
```

Comment:
This line feels like a red herring to me. K8s won't schedule the Redpanda Pod if it can't provide it with the # of CPUs it's requested. The issue of sharing cores would really only pop up if _not_ using the static CPU policy and additional Pods are being scheduled on the same node a redpanda.

Reply:
Removed.